aui.views/src/AUI/Image/AAnimatedDrawable.cpp

```diff
@@ -0,0 +1,21 @@
+//
+// Created by dervisdev on 1/12/2023.
+//
+
+#include "AAnimatedDrawable.h"
+
+void AAnimatedDrawable::draw(const IDrawable::Params &params) {
+    auto texture = Render::getNewTexture();
```

Comment:
New texture should not be created every frame. Consider to store the texture as a field of `AAnimatedDrawable`.